CMSgov/onemac
# OY2-20691 - update detail card format

Story: https://qmacbis.atlassian.net/browse/OY2-20691
Endpoint: See github-actions bot comment

### Details

Update Package Details page to add Status label above the current package status

### Changes

- add Review item surrounding status
- created css styles for status and for top level review items in a card (in order to manage whitespace)
- make package actions also a Review item to get consistent header labels

### Test Plan

1. Login as a state submitter
2. Navigate to Package List
3. Click any item in the list to view the Details page
4. Verify status label and package actions label look as specified
